### PR TITLE
chore(lvs/lvol): add trace message before zeroing super

### PR DESCRIPTION
**Pull request by tiagolobocastro**
This is useful to trace how long zeroing takes.
Also use write_zeroes variant that is now available which is much simpler.

### CODE REVIEW

**Review by hrudaya21 — APPROVED** [review]

File: io-engine/src/lvs/lvs_store.rs [hunk 1]
```diff
@@ -756,6 +756,8 @@ impl Lvs {
             })
             .map(Lvol::from_inner_ptr)?;
 
+        info!("{:?}: wipping super", lvol);
```
Comment (dsharma-dc) [inline io-engine/src/lvs/lvs_store.rs]: typo: `wiping`

**Review by dsharma-dc — APPROVED** [review]

File: io-engine/src/lvs/lvs_store.rs (hunk 1 above)
Comment (tiagolobocastro) [reply io-engine/src/lvs/lvs_store.rs]: good catch!

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**